<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCMESHTRANGUI_ReadMCNPFmeshDlg</class>
 <widget class="QDialog" name="MCMESHTRANGUI_ReadMCNPFmeshDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>195</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>568</width>
    <height>195</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Load File</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>531</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>MCNP Input and Meshtal file</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Input File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Meshtal File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rmf_le_inp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="rmf_le_meshtal">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="rmf_pb_inpbrw">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>96</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rmf_pb_meshtalbrw">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>96</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
